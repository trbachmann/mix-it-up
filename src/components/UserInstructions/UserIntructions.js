import React from 'react';
import { NavLink } from 'react-router-dom';
import blankRecipe from '../../images/kara-eads-unsplash.jpg';

export const UserInstructions = () => {
  return(
    <div className='UserInstructions--div'>
      <img className='UserInstructions-img' src={blankRecipe} alt='Empty recipe book and measuring cups'/>
      <h2>Start saving your recipes today!</h2>
      <p>Click on a recipe</p>
      <p>Add your personal baking notes and then save the recipe for future reference</p>
      <NavLink to='/' >
        <button className='UserInstructions--button'>Get started</button>
      </NavLink>
    </div>
  );
}