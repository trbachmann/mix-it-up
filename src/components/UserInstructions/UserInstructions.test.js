import React from 'react';
import { shallow } from 'enzyme';
import { UserInstructions } from './UserIntructions';

describe('UserInstructions', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<UserInstructions />);
    expect(wrapper).toMatchSnapshot();
  });
});