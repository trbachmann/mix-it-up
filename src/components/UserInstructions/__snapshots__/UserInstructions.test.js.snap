// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInstructions should match the snapshot 1`] = `
<div
  className="UserInstructions--div"
>
  <img
    alt="Empty recipe book and measuring cups"
    className="UserInstructions-img"
    src="kara-eads-unsplash.jpg"
  />
  <h2>
    Start saving your recipes today!
  </h2>
  <p>
    Click on a recipe
  </p>
  <p>
    Add your personal baking notes and then save the recipe for future reference
  </p>
  <NavLink
    activeClassName="active"
    aria-current="page"
    to="/"
  >
    <button
      className="UserInstructions--button"
    >
      Get started
    </button>
  </NavLink>
</div>
`;
